File: lib/api/me.js
var _ = require('lodash'),
  Promise = require('bluebird'),
  util = require('../util'),
  log = util.logger();

function Me(hootsuite, connection) {
  this._hootsuite = hootsuite;
  this._connection = connection;
}

Me.prototype = {
  get: function () {
    var path = util.createPath('me');
    return this._connection.get(path, { _method: 'GET' });
  },

  getOrganizations: function () {
    var path = util.createPath('me', 'organizations');
    return this._connection.get(path, { _method: 'GET' });
  },

  getSocialProfiles: function () {
    var path = util.createPath('me', 'socialProfiles');
    return this._connection.get(path, { _method: 'GET' });
  },
}

module.exports = Me;

File: src.net/hootsuite-rest/Api/Me.cs
using Newtonsoft.Json.Linq;

namespace Hootsuite.Rest.Api
{
    public class Me
    {
        Hootsuite _hootsuite;
        Connection _connection;

        public Me(Hootsuite hootsuite, Connection connection)
        {
            _hootsuite = hootsuite;
            _connection = connection;
        }

        public JToken get()
        {
            var path = util.createPath("me");
            return _connection.get(path);
        }

        public JToken getOrganizations()
        {
            var path = util.createPath("me", "organizations");
            return _connection.get(path);
        }

        public JToken getSocialProfiles()
        {
            var path = util.createPath("me", "socialProfiles");
            return _connection.get(path);
        }
    }
}
